tests/assets/ansible/roles/basepackages/tasks/Suse.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
I'm not sure if the `roles` structure makes sense. The whole `basepackages` and `repositories` roles are just specific to `CaaSP`. So from my point of view, it would make sense to have a single `caasp` role which has different tasks for setting up repos and install packages.

Reply:
The roles are written so that you can easily add any distro/k8s implementation.
For DISTRO you just add the DISTRO.yaml in the role and you're good to go. 

I can change to a dedicated role if you feel it is really better of course.

Reply:
I think one of the problems here is that for the upstream vanilla kubernetes on opensuse, we don't need these repositories. We might be better renaming "playbook_main" to "caasp" or something similar and only calling it on a caasp run.

Reply:
yeah. I would prefer to have a caasp role only for now. and change it when needed.

Reply:
Need to fix this one. Working on it

Reply:
Done